misc/extract_dtcs_and_components_from_excel_file.py

```diff
@@ -0,0 +1,129 @@
+import argparse
```

Comment:
Just a minor detail: I'd call the script `extract_knowledge_from_msi_table.py`. If we ever get another table from someone else, it's clear what we did. Additionally, we might be extracting more than DTCs and components at some point, e.g., `use_oscilloscope`.